EquestriaReapers/ER-frontend
# Epic/first frontend approach

## 🤔 Problem
We need to define an initial architecture for the project

## ✅ Solution
We have defined an initial architecture and implemented a couple of best practices
![image](https://github.com/luizzzito/ER-frontend/assets/22270579/1551ff6b-04f4-4b74-8594-1ea91229fc7a)

## 🔬 How to test
Proof that the login works!
- Go to /login
- Enter an email and a password that match one in the database
- You should be able to authenticate
